src/main/steps/c100-rebuild/applicant/add-applicants/postController.ts

```diff
@@ -40,35 +40,33 @@ export default class AddApplicantPostController extends PostController<AnyObject
     const NoOfApplicantLength = req.session.userCase.appl_allApplicants?.length;
     const checkIfApplicantLengthLess = NoOfApplicantLength === 0;
     const checkIfApplicantLengthLessAndFormError =
-      (checkIfApplicantLengthLess && applicantFirstName === '') ||
-      (checkIfApplicantLengthLess && applicantLastName === '');
+      checkIfApplicantLengthLess && (applicantFirstName === '' || applicantLastName === '');
     const saveAndComeBackToggled = req.body['saveAndComeLater'];
 
     if (saveAndComeBackToggled) {
       req.session.save();
       return super.saveAndComeLater(req, res, req.session.userCase);
+    }
+    if (saveAndContinueChecked) {
```

Comment:
Are these changes tested and working as expected?

Reply:
For a safer side this functionality is reverted back to original and also there are no complexity is found on this file.